test/views/html/employments/AddEmploymentPayrollNumberFormViewSpec.scala

```diff
@@ -62,9 +62,9 @@ class AddEmploymentPayrollNumberFormViewSpec extends TaiViewSpec with FormValues
           "tai.addEmployment.employmentPayrollNumber.error.selectOption")
         val formWithErrors: Form[AddEmploymentPayrollNumberForm] = AddEmploymentPayrollNumberForm.form
           .withError(AddEmploymentPayrollNumberForm.PayrollNumberChoice, noPayrollNumberChooseError)
-        def view: Html = add_employment_payroll_number_form(formWithErrors, payrollNumberViewModel)
+        val view = add_employment_payroll_number_form(formWithErrors, payrollNumberViewModel)
```

Comment:
just out of curiosity why was this changed to a val?

Reply:
because it's only called once and it's not reused outside the scope of this test